isis/src/base/apps/topds4/topds4.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<application name="topds4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
+  <brief>
+    Convert from ISIS3 cube to PDS 4 format using template XML files
+  </brief>
+
+  <description>
+    <p>
+      Writes a PDS4 compatible label file and image file. The contents of the label file are
+      generated useing the template file specified by the TEMPLATE parameter.
+    </p>
+    <p>
+      The image file is generated from the input ISIS cube specified by the FROM parameter.
+      The output image file will a copy of the input ISIS cube with the image data stored in

```

Comment:
missing a word in here. "The output image file will [contain?] a copy..."

Reply:
Just an FYI, the Inja link appears broken. However, I did find this one: https://github.com/pantor/inja.

Reply:
If this is the URL you mean, I am able to get to it: https://pantor.github.io/inja

Reply:
@scsides yep, I saw that link in the GitHub link I provided on the right side of the page. I clicked the link in email notification that highlights the URL and it sent me to a 404. Copy pasta worked fine. Once rendered by ISIS doc processing, I am sure it will be good.

I am interest in this app as a couple of our projects have phocube and projected map data that we need to archive to the PDS. I am hoping this app makes that process easier. 

Thanks...